logger.ts

```diff
@@ -5,6 +5,9 @@ import * as log4js from "log4js";
 import * as util from "util";
 import * as stream from "stream";
 import Future = require("fibers/future");
+import marked = require("marked");
+let TerminalRenderer = require("marked-terminal");
```

Comment:
Noted, however this module seems to be missing in [DefinatelyTyped](https://github.com/borisyankov/DefinitelyTyped) so writing down a .d.ts file should probably come as a latter PR